toolchain/sem_ir/inst_namer.cpp

```diff
@@ -953,14 +955,12 @@ auto InstNamer::CollectNamesInBlock(ScopeId top_scope_id,
         continue;
       }
       default: {
-        break;
+        // Sequentially number all remaining values.
+        if (untyped_inst.kind().has_type()) {
+          add_inst_name("");
+        }
       }
```

Comment:
Please add a `break` (or `continue`, if you keep all the `continue`s) here too, rather than falling off the bottom of the switch.

Reply:
Done